projects/core/components/alert/alert.component.ts

```diff
@@ -0,0 +1,121 @@
+import {AnimationOptions} from '@angular/animations';
```

Comment:
This is moved and changed, not new. We need to refactor it a bit before 3.0, keeping it this way to preserve backwards compatibility for context of notifications.